topologies/binding/options.go

```diff
@@ -23,13 +23,12 @@ import (
 	"google.golang.org/protobuf/proto"
 )
 
-// IANA assigns 9339 for gNxI, and 9559 for P4RT.  There hasn't been a
-// port assignment for gRIBI, so using Arista's default which is 6040.
+// IANA assigns 9339 for gNxI, 9559 for P4RT and 9340 for gRIBI.  
```

Comment:
I think gofmt is failing because there's whitespace at the end of this line. Please remove the trailing whitespace.

Reply:
yeah, fixed it in latest commit, sorry for that :)